request/builder.js

```diff
@@ -504,20 +503,20 @@ class ContractExecutionRequestBuilder {
     request.setCertVersion(this.certVersion);
     request.setFunctionArgument(this.functionArgument);
 
-    const contractIdEncoded = new TextEncoder('utf-8').encode(this.contractId);
+    const contractId = new TextEncoder('utf-8').encode(this.contractId);
```

Comment:
Why is it renamed ?
First of all, It's not consistent with the others, second, it doesn't seem to be changed.

Reply:
Sorry for that one, i think i will stick to **Encoded